### DISCUSSION TITLE
Geomap - draw polygon area from query

### DISCUSSION THREAD

**Discussion by featheredtoast** [grafana/grafana#60860] [Plugins]
So far it seems the only option to draw an area shape is from a static geoJSON file - It would be nice to have the option to query x,y coordinates from data, and draw a polygon area from those data points to allow for dynamic area displays.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
